Resources/Prototypes/Entities/Markers/Spawners/Random/posters.yml

```
@@ -138,10 +139,12 @@
       - PosterLegit50thAnniversaryVintageReprint
       - PosterLegitFruitBowl
       - PosterLegitPDAAd
+      - PosterLegitEnlist
       - PosterLegitNanomichiAd
       - PosterLegit12Gauge
       - PosterLegitHighClassMartini
       - PosterLegitTheOwl
+      #- PosterLegitNoERP # Einstein Engines: breaks literal 4th wall
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)